### PR TITLE
Remove js and css from index

### PR DESCRIPTION
**Pull request by madhu-lemosys**

### CODE REVIEW

File: index.htm [hunk 1]
```diff
@@ -71,28 +83,11 @@
         "familyName": "Anshakov",
         "sameAs": "http://alexey-anshakov.webrunes.com"
         },
-        "editor": "",
```
Comment (AlexeyAnshakov) [inline index.htm:74]: restore

File: index.htm [hunk 2]
```diff
@@ -1,27 +1,39 @@
 ﻿<!DOCTYPE html>
-<html lang="en-US">
```
Comment (AlexeyAnshakov) [inline index.htm:2]: Restore it

File: index.htm [hunk 3]
```diff
@@ -71,28 +83,11 @@
...
-        "comment": {
-        "@context": "http://schema.org",
-        "@type": "Comment",
-        "version": "575551107394633729"
-        }
+        "editor": ""
     }
     </script>
-
-    <!-- Temp Plus code begin -->
```
Comment (AlexeyAnshakov) [inline index.htm:83]: Restore it

File: index.htm [hunk 4]
```diff
@@ -71,28 +83,11 @@
...
-        "@type": "Plus",
-        "inLanguage": "en-US",
-        "name": ""
-        }
-    </script>
-    <!-- Temp Plus code end -->
-</head>
-
-<body>
-    <script src="http://wrio.s3-website-us-east-1.amazonaws.com/WRIO-InternetOS/WRIO.js"></script>
```
Comment (AlexeyAnshakov) [inline index.htm:96]: use this URL and name

**Comment by madhu-lemosys** [comment]
meta added

**Pull request merged**